unittests/FEXLinuxTests/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ ExternalProject_Add(FEXLinuxTests_32
 file(GLOB_RECURSE TESTS CONFIGURE_DEPENDS tests/*.cpp)
 file(GLOB_RECURSE TESTS_32_ONLY CONFIGURE_DEPENDS tests/*.32.cpp)
 
-set(TESTS_32 ${TESTS})
+set(TESTS_32 ${TESTS_32_ONLY})
```

Comment:
Could you make it so that the 32-bit-exclusive tests are treated symmetrically to the 64-bit-exclusive tests instead?

Conceptually there's no difference between the two AFAICT, so it's unclear now why there needs to be 32-bit specific logic here.

I don't fully understand what's going on the new version, but I'm guessing instead we'd want to have dedicated `TESTS_64_ONLY`/`TESTS_64` sets.

Reply:
Resolved. These are now treating 32-bit tests and 64-bit tests the same.
So these kind of fall in to three categories now, Tests that are supported on both 32-bit and 64-bit, tests that are supported on 64-bit only, and tests that are supported on 32-bit only.
In switching this over, it also uncovered a bug that we weren't running all the 32-bit tests, resulting in some test failures that cropped up due to a prologue being prepended on some 32-bit naked functions. Had to resolve that as well.

Reply:
Thanks, and great to have that issue with the 32-bit tests uncovered and resolved!